SwitchItem/SwitchItem.module.less

```diff
@@ -4,11 +4,48 @@
 @import "../styles/skin.less";
 
 .switchItem {
-	.switchIcon {
-		pointer-events: none;
+	.label {
+		vertical-align: middle;
+	}
+
+	.applySkins({
+		height: @agate-switchitem-height;
+		padding: @agate-switchitem-padding;
 
-		.applySkins({
+		.switchIcon {
+			pointer-events: none;
 			.margin-start-end(@agate-switch-icon-margin, 0);
+		}
+
+		.icon {
+			padding: @agate-switchitem-icon-padding;
+			margin: @agate-switchitem-icon-margin;
```

Comment:
I added `margin` for consistency with current implementation for all skins besides Silicon. 
![icon-margin](https://user-images.githubusercontent.com/63335823/114551329-ead20c00-9c6b-11eb-8207-c63c07b63422.PNG)

Indeed, it would have no effect if it's removed. Should we remove it?

Reply:
It will inherit the margin from Item. Yes, remove it.

Reply:
👍